src/Host/Client/Impl/Session/RSession.cs

```diff
@@ -73,9 +74,9 @@ internal sealed class RSession : IRSession, IRCallbacks {
             CanceledBeginInteractionTask = TaskUtilities.CreateCanceled<IRSessionInteraction>(new RHostDisconnectedException());
         }
 
-        public RSession(int id, IRHostConnector brokerConnector, Action onDispose) {
+        public RSession(int id, IRHostConnector hostConnector, Action onDispose) {
             Id = id;
-            BrokerConnector = brokerConnector;
+            RHostConnector = hostConnector;
             _onDispose = onDispose;
```

Comment:
Connector sounds like proxy pattern, maybe we should name it as such at some point. Not urgent.